src/Forms/Form.tsx

```diff
@@ -1,33 +1,51 @@
-import {
+import React, {
   FunctionComponent,
   FormEventHandler,
   FormEvent,
+  ReactNode,
+  ReactElement,
 } from 'react';
 import styled from 'styled-components';
 import { fromTheme } from '../Theme';
 
 export interface FormProps {
   /** The id of the form */
-  id?: string;
+  id: string;
   /** A label that specifies the purpose of the form */
   label: string;
   /** Handler attached to the onSubmit handler */
   submitHandler?: FormEventHandler<HTMLFormElement>
+  /** The form elements */
+  children: ReactNode[];
 }
 
-const Form: FunctionComponent<FormProps> = styled.form.attrs<FormProps>(
-  (props: FormProps) => ({
-    id: props.id,
-    'aria-label': props.label,
-    onSubmit: (e: FormEvent<HTMLFormElement>) => {
-      e.preventDefault();
-      props.submitHandler(e);
-    },
-  })
-)`
-  & > * {
-    margin: ${fromTheme('ws', 'small')};
-  }
+const StyledForm = styled.form`
+  margin: ${fromTheme('ws', 'small')};
 `;
 
+const Form: FunctionComponent<FormProps> = (props): ReactElement => {
```

Comment:
We can actually use `PropsWithChildren` here so that we don't have to type `children` directly in the `props` object

```suggestion
const Form: FunctionComponent<PropsWithChildren<FormProps>> = (props): ReactElement => {
```

Reply:
Ah sorry I had forgotten about this possibility. Thanks for pointing this out! (Fixed in [3f87a74](https://github.com/seas-computing/mark-one/pull/173/commits/3f87a742c75969f5fc9901fd5fdaf674c0d0bf7c))

Reply:
No worries!